src/main/java/seedu/duke/commands/ListModuleCommand.java

```diff
@@ -1,23 +1,23 @@
 package seedu.duke.commands;
 
 import seedu.duke.common.Messages;
-import seedu.duke.exceptions.CommandException;
+import seedu.duke.exception.CommandException;
 import seedu.duke.module.ModuleList;
 import seedu.duke.ui.UI;
 
-import static java.util.Objects.requireNonNull;
+import static seedu.duke.common.Messages.MESSAGE_MODULE_TO_LIST;

```

Comment:
Actually should this class be called ListModule**s**Command? To standardise with ListLesson**s**Command and ListTask**s**Command and also to be clearer

I think we just missed the **s** when typing